jobbergate-api/jobbergate_api/apps/applications/routers.py

```diff
@@ -227,18 +227,6 @@ async def application_delete(
     logger.debug(f"Preparing to delete {application_id=} from the database and S3")
 
     where_stmt = applications_table.c.id == application_id
-    get_query = applications_table.select().where(where_stmt)
```

Comment:
what happens if this route is called with an invalid id?

Reply:
There is a function a above taking care of it and returning 404 in this case:

```
application = await _fetch_application_by_id(application_id)
```

Reply:
This block was removed to avoid duplication